tests/test_repair.py

```diff
@@ -79,7 +79,7 @@ def test__find_intact_sub_trees():
     intact_sub_trees = obj._find_intact_sub_trees()
     basals = [section for section in intact_sub_trees
               if section.type == NeuriteType.basal_dendrite]
-    assert_equal(len(basals), 4)
+    assert_equal(len(basals), 78)
```

Comment:
that s a huge diff !
Why is it so big ?

Reply:
The code use to select the basal sections in the cut plane instead of ALL basals but the ones in the cut plane. No wonder why it is big :facepalm:

Reply:
ok!